src/openforms/prefill/contrib/haalcentraal/plugin.py

```diff
@@ -67,6 +67,8 @@ def _get_values_for_bsn(cls, bsn: str, attributes: Iterable[str]) -> Dict[str, A
     def get_prefill_values(
         cls, submission: Submission, attributes: List[str]
     ) -> Dict[str, Any]:
+        if not self.is_enabled:
+            raise PluginNotEnabled()
```

Comment:
This doesn't work right now, because `get_prefill_values` for HaalCentraal is implemented as a classmethod, I'm not sure if there is a specific reason for this?